packages/hoppscotch-cli/src/utils/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from "./constants";
+export * from "./checks";
+export * from "./request";
+export * from "./test";
+export * from "./getters";
+export * from "./mutators";
+export * from "./functions/array";
```

Comment:
Why are we exporting like this ?

Reply:
Actually, I currently import functions based on their dir path (like `import { .... } from /interface`). So, thats why I have exported the functions within /SOME_DIR to /SOME_DIR/index.ts

Reply:
Yeah gotcha, this is technically not a problem. But let's not do this.

The reason why I say we shouldn't is because of two reasons.
1. This encourages sensible file names and file structures.
2. The imports on the user side will have more contextual information about the thing they are importing. For example, in utils, we have various family of functions in different files each encomposing different behaviours. Now when we see the filename in the import, we get a more clear picture of what that function might do, for example, we know functions in `mutators` can mutate stuff (etc.)

We follow the opposite behaviour in library code though (you will see similar exports in `@hoppscotch/data` for example), the reasoning there is, when we are writing libraries, we shouldn't expose unwanted the internals of the library, we should only expose concerns. Since the CLI is not a library (or an intermediary product) and is a end product where there will be a lot more complex and cross-cutting concerns in the future, I like the code to be more explicit.